### ISSUE TITLE
Swap depth issue with `addmod` and `mulmod`

### ISSUE THREAD

**Issue by zemse** [Philogy/balls#5]
Input code:

```
fn HEY(x, y) -> (z) {
    z = addmod(x, y, 3)
}
```

Actual output:

```
#define macro HEY() = takes(2) returns(1) {
    // takes:                      [y, x]
    0x3                         // [y, x, 0x3]
    swap1                       // [y, 0x3, x]
    addmod                      // [z]
    // returns:                    [z]
}
```

Expected output:
```diff
#define macro HEY() = takes(2) returns(1) {
    // takes:                      [y, x]
    0x3                         // [y, x, 0x3]
-   swap1                       // [y, 0x3, x]
+   swap2                       // [y, x, 0x3]
    addmod                      // [z]
    // returns:                    [z]
}
```

**Comment by Philogy** [comment]
Thanks for reporting, have confirmed and will fix later this week.

**Comment by Will-Smith11** [comment]
@Philogy Crit in balls :sob:

**Comment by Philogy** [comment]
Fixed in df45004, thanks for reporting!
